recipes/staircase/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "staircase" %}
+{% set version = "1.5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ded45fb4444bb5273cee6ca13d092952a908c72968eef0c26776996adae4a1ec
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv
```

Comment:
The current recommended `script` command is:

```
script: "{{ PYTHON }} -m pip install . -vv"
```

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35